.github/workflows/build.yml

```diff
@@ -238,7 +238,7 @@ jobs:
           name: artifact
           path: dist
 
-      - uses: pypa/gh-action-pypi-publish@v1
+      - uses: pypa/gh-action-pypi-publish@v1.5.0
```

Comment:
Apparently, they don't do the proper thing to keep a v1 tag.

I wanted to keep it open so that it picks the latest since this version is already one and a half years old.

Next time we will update to the latest tag, since they are using an old depreacted version of node in 1.5.0